app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -116,6 +118,7 @@
     private int pokemonHP;
     private boolean pokeFlyRunning = false;
     private int trainerLevel;
+    private LruCache<String, String> OCRCache = new LruCache<>(200);
```

Comment:
1. `OCRCache` is a class name—field names should be lowercase by conventions, so `ocrCache`.
2. Mapping from String is questionable.